### PR TITLE
Mrh OnlineMusicList Page

### PR DESCRIPTION
**Pull request by mahdi302**
This page is as same as music List page but its uses for shows Online kinds of music

### CODE REVIEW

File: app/src/main/java/com/worldsnas/starplayer/view/online_music_list/OnlineMusicListFragment.kt [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import com.worldsnas.starplayer.databinding.FragmentOnlineMusicsListBinding
+import com.worldsnas.starplayer.di.DaggerOnlineMusicListComponent
+import com.worldsnas.starplayer.di.OnlineMusicListComponent
+import com.worldsnas.starplayer.model.LocalMusic
+import com.worldsnas.starplayer.model.Music
+import kotlinx.android.synthetic.main.fragment_online_musics_list.view.*
+import javax.inject.Inject
+
+/**
+ * created by Roohandeh  4/29/2020
```
Comment (helloseyedjafari) [inline app/src/main/java/com/worldsnas/starplayer/view/online_music_list/OnlineMusicListFragment.kt]: pak kon lotfan

File: app/src/main/java/com/worldsnas/starplayer/view/online_music_list/OnlineMusicListFragment.kt [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+    private val TAG = OnlineMusicListFragment::class.java.simpleName
+    private var _binding: FragmentOnlineMusicsListBinding? = null
+    private val binding get() = _binding!!
+    private lateinit var onlineMusicListComponent: OnlineMusicListComponent
+    private val adapter =
+        OnlineMusicListAdapter { item -> onlineMusicListListener(item) }
+
+    @Inject
+    lateinit var onlineMusicListViewModel: OnlineMusicListViewModel
```
Comment (helloseyedjafari) [inline app/src/main/java/com/worldsnas/starplayer/view/online_music_list/OnlineMusicListFragment.kt]: `ViewModel` injori nabayad sakhte beshe

File: app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -1,4 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="fragment_padding">16dp</dimen>
+
+    <dimen name="eight_dp">8dp</dimen>
```
Comment (helloseyedjafari) [inline app/src/main/res/values/dimens.xml]: esm haye behtar bezar lotfan

**Review by helloseyedjafari — CHANGES_REQUESTED** [review]
you could choose a better branch name
good job small changes and integerate with repo and we are done here

File: app/src/main/java/com/worldsnas/starplayer/view/online_music_list/OnlineMusicListFragment.kt (hunk 1 above)
Comment (mahdi302) [reply app/src/main/java/com/worldsnas/starplayer/view/online_music_list/OnlineMusicListFragment.kt]: Kole class ro ?

File: app/src/main/java/com/worldsnas/starplayer/view/online_music_list/OnlineMusicListFragment.kt (hunk 2 above)
Comment (IamHaddad) [reply app/src/main/java/com/worldsnas/starplayer/view/online_music_list/OnlineMusicListFragment.kt]: tip : viewmodel module ro bbinid

File: app/src/main/java/com/worldsnas/starplayer/view/online_music_list/OnlineMusicListFragment.kt (hunk 1 above)
Comment (IamHaddad) [reply app/src/main/java/com/worldsnas/starplayer/view/online_music_list/OnlineMusicListFragment.kt]: comment ro pak konid

File: app/src/main/java/com/worldsnas/starplayer/view/online_music_list/OnlineMusicListFragment.kt (hunk 2 above)
Comment (mehdirouhandeh) [reply app/src/main/java/com/worldsnas/starplayer/view/online_music_list/OnlineMusicListFragment.kt]: ok

**Comment by mahdi302** [comment]
taghirate gofte shode anjam shod. age dobare moshkeli blood begid. 5.2.2020

**Pull request closed without merging**